llvm/test/tools/llvm-debuginfo-analyzer/WebAssembly/Inputs/pr-44884-clang.yaml

```diff
@@ -0,0 +1,241 @@
+    Payload:         020000006B0000006D0000003F0100000000000000000000
+  - Type:            CUSTOM
+    Name:            .debug_str
+    Payload:         636C616E672076657273696F6E2031392E302E3067697420282F646174612F70726F6A656374732F6C6C766D2D726F6F742F6C6C766D2D70726F6A6563742F636C616E672032646236373033663063323537643239336466343535653264666638633166623639356334313030290070722D34343838342E637070002F646174612F70726F6A656374732F736372697074732F72656772657373696F6E2D73756974652F696E7075742F67656E6572616C00696E74005F5A336261726600626172005F5A33666F6F6300666F6F00756E7369676E656420696E7400496E70757400666C6F617400506172616D00636861720056616C756500494E5400416464656400464C4F415400
+  - Type:            CUSTOM
+    Relocations:
+      - Type:            R_WASM_FUNCTION_OFFSET_I32
+        Index:           0
+        Offset:          0x31
+      - Type:            R_WASM_FUNCTION_OFFSET_I32
+        Index:           2
+        Offset:          0x4D
+    Name:            .debug_line
+    Payload:         8A000000040024000000010101FB0E0D0001010101000000010000010070722D34343838342E6370700000000000000502020000000105240A08F20602321121B92105184A02040001010005026D00000014050F0A022914050706088205150678051D0674051B0882051574050D5805130675050F0674050D9E050A067705120674051008820503740217000101
+  - Type:            CUSTOM
+    Name:            linking
+    Version:         2
+    SymbolTable:
+      - Index:           0
+        Kind:            FUNCTION
+        Name:            _Z3barf
+        Flags:           [ VISIBILITY_HIDDEN ]
+        Function:        0
+      - Index:           1
+        Kind:            GLOBAL
+        Name:            __stack_pointer
+        Flags:           [ UNDEFINED ]
+        Global:          0
+      - Index:           2
+        Kind:            FUNCTION
+        Name:            _Z3fooc
+        Flags:           [ VISIBILITY_HIDDEN ]
+        Function:        1
+      - Index:           3
+        Kind:            SECTION
+        Flags:           [ BINDING_LOCAL ]
+        Section:         4
+      - Index:           4
+        Kind:            SECTION
+        Flags:           [ BINDING_LOCAL ]
+        Section:         6
+      - Index:           5
+        Kind:            SECTION
+        Flags:           [ BINDING_LOCAL ]
+        Section:         7
+      - Index:           6
+        Kind:            SECTION
+        Flags:           [ BINDING_LOCAL ]
+        Section:         8
+  - Type:            CUSTOM
+    Name:            producers
+    Languages:
+      - Name:            C_plus_plus_14
+        Version:         ''
+    Tools:
+      - Name:            clang
+        Version:         '19.0.0git (/data/projects/llvm-root/llvm-project/clang 2db6703f0c257d293df455e2dff8c1fb695c4100)'
+  - Type:            CUSTOM
+    Name:            target_features
+    Features:
+      - Prefix:          USED
+        Name:            mutable-globals
+      - Prefix:          USED
+        Name:            sign-ext
+...
```

Comment:
Its really up to you, but I think the the .s format might be readable and editable here.  But it if you are always planning on using the `.cpp` as the source of truth I guess it doesn't matter so much?

I guess maybe using the `.s` format means depending on a little more of llvm?

If the .cpp files *are* the source of truth here perhaps include a readme/script for how to regenerate the yaml files?